packages/server/src/responder/component.ts

```diff
@@ -21,37 +21,37 @@ export enum ResponderStateKind {
 export type PendingResponseState = {
     appointmentId: string;
     kind: ResponderStateKind.Pending;
-    identifier: PisaTransactionIdentifier;
+    identifier: PisaTransactionIdentifierSerialisation;
```

Comment:
This, and below, no longer needs to be the `PisaTransactionIdentifierSerialisation` it can just be `PisaTransactionIdentifier`

Reply:
Done in 7fd4c588bf8ecc05f46bf446231edf8a53acdc3b, together with a larger refactoring of all the types (anchor states) using PisaTransactionIdentifier.

Also, in 349855d6eacb4a9918091b336a60b168bf9de1ba I added all the deserialisers to the service. This might require attention, as it would only be detected at runtime when we deserialise the db, and we currently don't have anything testing for that.